src/cloudformation_cli_python_lib/log_delivery.py

```diff
@@ -43,7 +43,7 @@ def setup(
             stream_name = f"{request.awsAccountId}-{request.region}"
 
         log_handler = cls._get_existing_logger()
-        if provider_sess and log_group:
+        if provider_sess and log_group and request.resourceType:
```

Comment:
is it checking if the type is `None` or empty or both?

Reply:
Both. From ipython in development environment:
```
In [1]: str_none = None
In [2]: str_empty = ""
In [3]: if str_empty or str_none:
   ...:     print("one of these is not like the other")
   ...: else:
   ...:     print("same")
   ...: 
same

```